packages/fxa-auth-client/lib/client.ts

```diff
@@ -540,6 +540,7 @@ export default class AuthClient {
       marketingOptIn?: boolean;
       newsletters?: string[];
       style?: string;
+      scopes?: string[];
```

Comment:
micro-nit: if you put this field next to the existing `service` field it might make it slightly more obvious that they're kind of similar (in that they both indicate what the requested session is going to be used for).